test/ui-testing/tags-crud.js

```diff
@@ -0,0 +1,98 @@
+/* global describe, it, before, after, Nightmare */
+const AgreementCRUD = require('./agreement-crud');
+
+module.exports.test = (uiTestCtx) => {
+  const number = Math.round(Math.random() * 100000);
+  const testTag = `test${Math.floor(Math.random() * 10000)}`;
+  const agreement = {
+    name: `Tags Agreement #${number}`,
+  };
+
+  describe('Login > Enable Tags > Find user > Create Tags > Logout\n', function test() {
+    const { config, helpers } = uiTestCtx;
+    const nightmare = new Nightmare(config.nightmare);
+    nightmare.options.width = 1000; // added this temporarily as MultiSelect doesnt work well with narrow screen sizes
+
+    this.timeout(Number(config.test_timeout));
+
+    describe('login > create license > edit terms > create agreement > link license > check term > logout', () => {
+      before((done) => {
+        helpers.login(nightmare, config, done);
+      });
+
+      after((done) => {
+        helpers.logout(nightmare, config, done);
+      });
+
+      it('should Enable tags in settings', done => {
+        nightmare
+          .wait(config.select.settings)
+          .click(config.select.settings)
+          .wait('#app-list-item-clickable-settings')
+          .wait('a[href="/settings/tags"]')
+          .click('a[href="/settings/tags"]')
+          .wait('a[href="/settings/tags/general"]')
+          .click('a[href="/settings/tags/general"]')
+          .wait('#tags_enabled')
+          .wait(222)
+          .evaluate(() => {
+            const list = document.querySelectorAll('[value="true"]');
```

Comment:
Can we change this to `#tags_enabled[value="true"]`

Reply:
Updated